src/lib/viewers/BaseViewer.js

```diff
@@ -767,9 +767,16 @@ class BaseViewer extends EventEmitter {
      * @param {string} mode - Target annotation mode
      * @return {Function|null} Click handler
      */
-    /* eslint-disable no-unused-vars */
-    getAnnotationModeClickHandler(mode) {}
-    /* eslint-enable no-unused-vars */
+    getAnnotationModeClickHandler(mode) {
+        if (!mode || !this.isAnnotatable(mode)) {
```

Comment:
It defaults to the value of previewOptions.showAnnotations if no viewer specific options are specified